tests/cupy_tests/cuda_tests/test_stream.py

```diff
@@ -60,10 +63,12 @@ def test_eq_hip(self):
 
     def check_del(self, null, ptds):
         stream = cuda.Stream(null=null, ptds=ptds).use()
+        assert stream is cuda.get_current_stream()
         stream_ptr = stream.ptr
         x = from_data.array([1, 2, 3])
         del stream
-        assert cuda.Stream.null == cuda.get_current_stream()
+        cuda.Stream.null.use()
```

Comment:
Should we test that the stream is still alive after deletion?
```suggestion
        assert stream is cuda.get_current_stream()
        cuda.Stream.null.use()
```